OpenKeychain/src/main/java/org/sufficientlysecure/keychain/ui/SecurityTokenOperationActivity.java

```diff
@@ -183,6 +185,13 @@ protected void doSecurityTokenInBackground() throws IOException {
 
         switch (mRequiredInput.mType) {
             case SECURITY_TOKEN_DECRYPT: {
+                long tokenKeyId = KeyFormattingUtils.getKeyIdFromFingerprint(
+                        mSecurityTokenHelper.getMasterKeyFingerprint(KeyType.SIGN));
```

Comment:
Ah SIGN is the first key which is also the master key correct?
Is the method ``getMasterKeyFingerprint`` named properly? Should it be named ``getKeyFingerprint``? I know it's not your fault :P